source/database/src/data/sql/nature2021/dataset_metadata.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO nature2021.dataset_metadata (key, value) VALUES ('MINIMUM_YEAR', '2018');
+INSERT INTO nature2021.dataset_metadata (key, value) VALUES ('MAXIMUM_YEAR', '2030');
```

Comment:
Wasn't the maximum 2040 for nature2021?

Reply:
I heard it would be 2035, also heard is would be 2040, so left it as it is...